task5/src/main/bsu/com/ForwardServlet.java

```diff
@@ -0,0 +1,14 @@
+package main.bsu.com;
```

Comment:
Cosmetic: Source root is usually begins in src/main or exactly in src/main/java so the main point is that we do not use 'main' as a package.